overlays/emscripten.nix

```diff
@@ -2,6 +2,8 @@ final: prev: let
   # GHCJS currently requires an old version of emscripten. Overriding to build the older version doesn't
   # work straightforwardly on 20.09 anymore, so we use an ugly hack and pull emscripten from 20.03.
   # In future, we'll just fix GHCJS to work with the newer emscripten.
+
+  # Not sure what to do here - akrmn
```

Comment:
hey @michaelpj, I think I got rid of all occurrences of `nixpkgs-2003` or `R2003`, except for this one. The comment above mine is a bit worrying.

Reply:
Yep, this is a bit worrying. I also don't know what to do about this.

Reply:
We will need to keep a nixpkgs-2003 pin for now